docs/plugin-development.rst

```diff
@@ -389,6 +389,18 @@ You can use all of the testing utilities provided by the ``base.TestCase`` class
 from core. You will also get coverage results for your plugin aggregated with
 the main Girder coverage results if coverage is enabled.
 
+Plugins can also use the external data interface provided by Girder as described
+in :ref:`use_external_data`.  For plugins, the data key files should be placed
+inside a directory called ``plugin_tests/data/``.  When referencing the
+files, they must be prefixed by your plugin name as follows
+
+.. code-block:: cmake
+
+    add_python_test(my_test EXTERNAL_DATA plugins/cats/test_file.txt)
```

Comment:
How do you feel about doing some more work in the [``file(COPY ${_base_data_files}``](https://github.com/girder/girder/pull/1214/files#diff-2b950cd7aa99f78420ee28f59a825688R139) step and/or the [``list(APPEND _data_files ``](https://github.com/girder/girder/pull/1214/files#diff-2b950cd7aa99f78420ee28f59a825688R139) to (only if the test is for a ``PLUGIN``) attempt to search / glob for files in the ``/tests/data/`` vs ``/tests/data/plugins/<name>/`` and add the appropriate prefix to each data key file?